HousingSearchApi.Tests/V1/E2ETests/Steps/GetAssetSteps.cs

```diff
@@ -74,6 +78,20 @@ public async Task WhenNoOfBedSpacesIsProvided(int numberOfBedSpaces)
             _lastResponse = await _httpClient.GetAsync(route).ConfigureAwait(false);
         }
 
+        public async Task WhenContractIsApprovedIsProvided(string contractApprovalStatus)
+        {
+            var route = new Uri($"api/v1/search/assets/all?contractIsApproved={contractApprovalStatus}&page={1}",
+                UriKind.Relative);
+
+            _lastResponse = await _httpClient.GetAsync(route).ConfigureAwait(false);
+        }
+
+        public async Task WhenContractIsApprovedIsNotProvided()
```

Comment:
Not a big deal as it is following the existing pattern, but shouldn't there just be a `WhenNoQueryParametersAreProvided` method for these instead?